sdk/Trace/Event.php

```diff
@@ -13,11 +12,10 @@ class Event implements API\Event
     private $timestamp;
     private $attributes;
 
-    // todo: pick datatype for timestamp
-    public function __construct(string $name, ?API\Attributes $attributes = null, string $timestamp = null)
+    public function __construct(string $name, ?API\Attributes $attributes = null, int $timestamp = null)
     {
         if (null === $timestamp) {
-            $timestamp = (new Clock())->millitime();
+            $timestamp = hrtime(true);
```

Comment:
It makes sense for events to use timestamps, not durations -- these would be `microtime`s, right?

Reply:
They would be.  Sorry for my mess here, hoping to clean it up today.